### ISSUE TITLE
Empty repost in Conversation view

### ISSUE THREAD

**Issue by poweruser82** [jeena/Bungloo#129]
I have an empty post in conversation:
![Schermata 2013-01-17 alle 19 01 13](https://f.cloud.github.com/assets/325193/76581/aa3e8994-60f5-11e2-9388-02e42959b753.png)
Looking at the profile view and at [TentStatus](https://andybroomfield.tent.is/posts/IAAI3wT2JJVn5PeinwM0eQ), it looks like a repost.
![Schermata 2013-01-17 alle 23 23 47](https://f.cloud.github.com/assets/325193/76583/d8e1b26c-60f5-11e2-9fad-d6636036f71e.png)
Console says nothing except

```
17/01/13 23:27:00,456 Tentia[13462]: jsa<TimelineView>: ERROR: getURL (404)Not Found GET (https://poweruser.tent.is/tent/followings/https%3A%2F%2Fandybroomfield.tent.is): '{"error":"Not Found"}'
```

**Comment by poweruser82** [comment]
![Schermata 2013-01-23 alle 22 39 31](https://f.cloud.github.com/assets/325193/91571/7bbdceea-65a5-11e2-95e9-15e840e47cb4.png)

**Comment by jeena** [comment]
That happens if the original post has been removed as far as I remember.

**Comment by jeena** [comment]
Ah but it only happens in the conversation view, I see.

**Comment by poweruser82** [comment]
Kevin's post is available on TentStatus https://kevinkleinman.tent.is/posts/RIECovEUQBTnzPm3e_6HxQ

**Comment by jeena** [comment]
This should be fixed now in 2.0
